client/java/src/main/java/com/linecorp/centraldogma/client/Latest.java

```diff
@@ -30,13 +34,14 @@
 public final class Latest<U> {
 
     private final Revision revision;
+    @Nullable
     private final U value;
 
     /**
      * Creates a new instance with the specified {@link Revision} and value.
      */
-    public Latest(Revision revision, U value) {
-        this.revision = revision;
+    public Latest(Revision revision, @Nullable U value) {
+        this.revision = requireNonNull(revision, "revision");
```

Comment:
`value` is Nullable but requireNonNull? Could you remove this validation?

Reply:
This `requireNonNull` is for `revision`. :sweat_smile: